src/main/java/com/mentoring/ecommerce/adapter/in/web/request/ProductRequest.java

```diff
@@ -7,14 +7,46 @@
 import javax.validation.constraints.NotBlank;
 import java.math.BigDecimal;
 
-@Data
+//@Data
```

Comment:
**Problem:** `response` body returning empty objects.

**Root Cause:** `response` and `request` mapping depends on `getters` and `setters`. But for some reason, _Lombok_ `@Data` was failing to generate them very well. When replaced with manual getters and setters, it worked as expected.

My guess: maybe I didn't install/configure _Lombok_ correctly.